### ISSUE TITLE
Fcutoff can be set beyond chip capability

### ISSUE THREAD

**Issue by tfcollins** [analogdevicesinc/ad936x-filter-wizard#26]
If you manually select a Fcutoff this will recalculate caldiv and can choose an unusable setting.  For example putting in 2000 will produce 108.4468 (caldiv==1 PLL==983040000), and the chip is limited to 39.2 on the LPF.  We just need to add this check, rfbw calculation already checks this.

This kinda breaks the design flow of manually selecting Fcutoff, but we support it at the moment.

**Comment by rgetz** [comment]
Actually - it's caldiv that sets things - if it can do it, it's likely the chip works fine.

**Comment by tfcollins** [comment]
Yes the chip will work fine, its just our model of the analog filters is off.

**Comment by rgetz** [comment]
All I'm saying is the chip is not limited to 39.2 on the low pass filter. The filter structure is limited only by caldiv.

-Robin
